acceptance/acceptance_test.go

```diff
@@ -229,1905 +99,1417 @@ func testWithoutSpecificBuilderRequirement(
-			h.AssertNil(t, err)
-			h.AssertNil(t, packageTomlFile.Close())
-
-			return packageTomlFile.Name()
-		}
-
 		when("no --format is provided", func() {
 			it("creates the package as image", func() {
 				packageName := "test/package-" + h.RandString(10)
-				output, err := h.RunE(subjectPack("package-buildpack", packageName, "-p", filepath.Join(tmpDir, "package.toml")))
-				h.AssertNil(t, err)
-				h.AssertContains(t, output, fmt.Sprintf("Successfully created package '%s'", packageName))
-				defer h.DockerRmi(dockerCli, packageName)
 
-				assertImageExistsLocally(packageName)
+				output := pack.SuccessfulRunWithOutput(
+					"package-buildpack", packageName,
+					"-p", simplePackageConfigPath,
+				)
+
+				assert.Contains(output, fmt.Sprintf("Successfully created package '%s'", packageName))
+				defer containerManager.RemoveImages(packageName)
+
+				assert.ImageExistsLocally(packageName)
 			})
-		})
 
-		when("--format image", func() {
-			it("creates the package", func() {
+			it("creates the package as a local image", func() {
 				t.Log("package w/ only buildpacks")
-				nestedPackageName := packageBuildpackLocally(filepath.Join(tmpDir, "package.toml"))
-				defer h.DockerRmi(dockerCli, nestedPackageName)
-				assertImageExistsLocally(nestedPackageName)
-
-				t.Log("package w/ buildpacks and packages")
-				aggregatePackageToml := generateAggregatePackageToml("simple-layers-parent-buildpack.tgz", nestedPackageName)
-				packageName := packageBuildpackLocally(aggregatePackageToml)
-				defer h.DockerRmi(dockerCli, packageName)
-				assertImageExistsLocally(packageName)
+				nestedPackageName := newPackageName()
+				assert.PackageCreated(
+					nestedPackageName,
+					pack.PackageBuildpack(nestedPackageName, simplePackageConfigPath),
+				)
+				defer containerManager.RemoveImages(nestedPackageName)
+
+				assert.ImageExistsLocally(nestedPackageName)
+
+				aggregatePackageConfigFile := pack.AggregatePackageFixture(
+					nestedPackageName,
+					"simple-layers-parent-buildpack.tgz",
+					tmpDir,
+				)
+
+				packageName := newPackageName()
+				assert.PackageCreated(packageName, pack.PackageBuildpack(packageName, aggregatePackageConfigFile.Name()))
+
+				defer containerManager.RemoveImages(packageName)
+				assert.ImageExistsLocally(packageName)
 			})
 
 			when("--publish", func() {
 				it("publishes image to registry", func() {
-					nestedPackageName := packageBuildpackRemotely(filepath.Join(tmpDir, "package.toml"))
-					defer h.DockerRmi(dockerCli, nestedPackageName)
-					aggregatePackageToml := generateAggregatePackageToml("simple-layers-parent-buildpack.tgz", nestedPackageName)
+					nestedPackageName := registryConfig.RepoName(newPackageName())
+					assert.PackagePublished(
+						nestedPackageName,
+						pack.PackageBuildpack(nestedPackageName, simplePackageConfigPath, "--publish"),
+					)
+					defer containerManager.RemoveImages(nestedPackageName)
 
```

Comment:
```suggestion
assert.ImageExistsOnlyInRegistry(nestedPackageName, registryConfig)
```

Reply:
We don't currently test this, so again, in service of the goal of achieving a like for like rewrite I'd prefer to add this in another PR.